ci/pipeline-backup-metadata.yml

```diff
@@ -104,160 +159,90 @@ jobs:
     privileged: true
-    file: ci/ci/oci/build-with-pack.yml
+    file: capi-ci/ci/oci/build-with-pack.yml
     params:
       CONTEXT_PATH: src/backup-metadata
       ADDITIONAL_ARGS: "--buildpack gcr.io/paketo-buildpacks/go --buildpack gcr.io/paketo-buildpacks/procfile --default-process wait"
     input_mapping:
       source-repository: src
-      capi-ci: ci
   - task: run-deplab
     privileged: true
-    file: ci/ci/docker/run-deplab.yml
+    file: capi-ci/ci/docker/run-deplab.yml
     input_mapping:
       source-repository: src
   - put: backup-metadata-docker-image
     params:
       image: labeled-image/image.tar
 
-- name: bump-image-sha
-  plan:
-    - in_parallel:
-      - get: src
-        passed: [build-backup-metadata-image]
-      - get: capi-k8s-release
-      - get: ci
-      - get: backup-metadata-docker-image
-        trigger: true
-    - task: bump-image-sha
-      file: ci/ci/backup-metadata/bump-image-sha.yml
-    - put: capi-k8s-release
-      params:
-        repository: updated-capi-k8s-release
-        rebase: true
-
-- name: build-artifact
-  plan:
-    - in_parallel:
-      - get: ci
-      - get: capi-k8s-release
-        trigger: true
-      - get: backup-metadata-docker-image
-        passed: [bump-image-sha]
-      - get: version
-        params: {pre: build}
-    - task: build-artifact
-      config:
-        platform: linux
-        image_resource:
-          type: docker-image
-          source:
-            repository: pcfplatformrecovery/backup-and-restore-cf-for-k8s
-        inputs:
-          - name: capi-k8s-release
-          - name: version
-        outputs:
-          - name: tas-cf-metadata-test-artifacts-capi
-        run:
-          dir: capi-k8s-release/src/backup-metadata
-          path: sh
-          args:
-          - "-c"
-          - "make artifact && mv cf-metadata.tgz ../../../tas-cf-metadata-test-artifacts-capi/backup-metadata.$(cat ../../../version/number).tgz"
-    - put: version
-      params: { file: version/number }
-    - put: tas-cf-metadata-test-artifacts-capi
-      params:
-        file: tas-cf-metadata-test-artifacts-capi/*.tgz
-
-- name: create-cluster
-  plan:
-  - in_parallel:
-    - get: tas-cf-metadata-test-artifacts-capi
-      passed: [build-artifact]
-      trigger: true
-    - get: capi-k8s-release
-      passed: [build-artifact]
-    - get: ci
-  - in_parallel:
-    - task: create
-      file: ci/ci/backup-metadata/create-cluster.yml
-      params:
-        GKE_SERVICE_ACCOUNT_KEY: ((backup_metadata_gke_service_account_key))
-        CLUSTER_PREFIX: ci-cluster
-    - task: create-velero-bucket
-      file: ci/ci/backup-metadata/create-velero-bucket.yml
-      params:
-        GKE_SERVICE_ACCOUNT_KEY: ((backup_metadata_gke_service_account_key))
-  - in_parallel:
-    - put: cluster-name
-      params:
-        file: cluster-name/info
-    - put: velero-bucket
-      params:
-        file: velero-bucket/info
-
-- name: install-cf-metadata-and-test
```

Comment:
For my own understanding, you said this step is unnecessary anymore because it's installed by default in CF now; where exactly is that? Is that from the following where we added the metadata generator container to one of our deployments: https://github.com/cloudfoundry/capi-k8s-release/blob/71a03ba8f4ad1800af028073e9b656490de12516/templates/controllers_deployment.yml#L53-L78?

Reply:
Yep, that was done in this PR: https://github.com/cloudfoundry/capi-k8s-release/pull/97

We no longer have to `kubectl apply` this separately and have separate secrets/config for it.